site/src/request_handlers/github.rs

```diff
@@ -138,13 +137,30 @@ async fn handle_rust_timer(
         return Ok(github::Response);
     }
 
-    for captures in build_captures(&comment.body).map(|(_, captures)| captures) {
-        let include = captures.get(2).map(|v| v.as_str());
-        let exclude = captures.get(3).map(|v| v.as_str());
-        let runs = captures.get(4).and_then(|v| v.as_str().parse::<i32>().ok());
```

Comment:
this really does make me wonder why our runs are i32s...

Reply:
I'm pretty sure it's a remnant of PostgreSQL - it requires you to use signed integers, or at least the Rust binding that we use.